src/components/molecules/card/CardHeader.tsx

```diff
@@ -40,6 +60,19 @@ const CardHeader: FC<IProps> = ({ variant, text, road }) => {
         </Box>
       );
       break;
+    case HeaderVariant.Label:
+      headerContent = (
+        <Box display="flex" alignItems="right" flex={1} className={classes.labelWrapper}>
+          <Box className={classes.roadSVG}>
+            <RoadNumberImage roadNumber={road} />
+          </Box>
+          <Box textAlign="center" whiteSpace="normal" className={classes.labelTypography}>
```

Comment:
why `whiteSpace="normal"`?

Reply:
I forgot the `normal` value is the initial value of white-space.
i will remove it .